diff --git a/stanford_alpaca/requirements.txt b/stanford_alpaca/requirements.txt
index f6cb386..85e88a6 100644
--- a/stanford_alpaca/requirements.txt
+++ b/stanford_alpaca/requirements.txt
@@ -6,4 +6,4 @@ transformers>=4.28.1
 torch
 sentencepiece
 tokenizers>=0.13.3
-wandb
+wandb
\ No newline at end of file
diff --git a/stanford_alpaca/train.py b/stanford_alpaca/train.py
index ed6f18c..6e75869 100644
--- a/stanford_alpaca/train.py
+++ b/stanford_alpaca/train.py
@@ -121,6 +121,7 @@ def preprocess(
     input_ids = examples_tokenized["input_ids"]
     labels = copy.deepcopy(input_ids)
     if (not data_args.train_wo_ignore_index):
+        print('Imma hereeee')
         for label, source_len in zip(labels, sources_tokenized["input_ids_lens"]):
             label[:source_len] = IGNORE_INDEX
     if data_args.train_wo_outputs:
